src/ucp/core/ucp_context.c

```diff
@@ -2044,6 +2060,19 @@ static ucs_status_t ucp_fill_config(ucp_context_h context,
             ((context->config.ext.fence_mode == UCP_FENCE_MODE_AUTO) &&
              (context->config.ext.max_rma_lanes > 1));
 
+    if (context->config.ext.odp_reg_mem_types & UCS_BIT(UCP_ODP_MEM_REG_TYPES_AUTO)) {
+        if (context->config.ext.odp_reg_mem_types != UCS_BIT(UCP_ODP_MEM_REG_TYPES_AUTO)) {
+            ucs_error("If UCP_ODP_MEM_REG_TYPES contains 'auto' then it "
+                      "shouldn't contain other memory types");
+        }
+    } else {
```

Comment:
Do we allow an empty configuration parameter to be used?

Reply:
As far as I know, yes, we do. But this API isn't finalized yet, so maybe it will be changed in the future.

Reply:
Is it ok that `MEMTYPE_REG_NB_TYPES=auto` is the same as `MEMTYPE_REG_NB_TYPES=`?

Reply:
At this moment, yes. It is going to be different in the one of future configurations.